src/Form/Form.php

```diff
@@ -245,6 +245,37 @@ public function cleanDBonPurge()
         );
     }
 
+    /**
+     * Give cron information
+     *
+     * @param string $name  Task's name
+     *
+     * @return array Array of information
+     **/
+    public static function cronInfo($name)
+    {
+        return ['description' => __('Purge old form drafts')];
+    }
+
+    /**
+     * Cron action to purge old form drafts
+     *
+     * @param CronTask|null $task
+     *
+     * @return int
+     * @used-by CronTask
+     */
+    public static function cronPurgeDraftForms($task = null)
+    {
+        $form = new Form();
+        $form->deleteByCriteria([
+            'is_draft' => 1,
+            'date_mod' => ['<', date('Y-m-d H:i:s', strtotime('-1 day'))],
```

Comment:
I would prefer a few more days to be extra safe (original code was 7 days).

```suggestion
            'date_mod' => ['<', date('Y-m-d H:i:s', strtotime('-7 day'))],
```